src/utils/internal/getCallFrame.ts

```diff
@@ -2,25 +2,22 @@
  * Return the stack trace frame of a function's invocation.
  */
 export function getCallFrame() {
-  try {
-    const inspectionError = new Error()
-    inspectionError.name = 'Inspection Error'
-    throw inspectionError
-  } catch (error) {
-    const frames: string[] = error.stack.split('\n')
+  const frames: string[] = (new Error().stack as string).split('\n')
 
-    // Get the first frame that doesn't reference the library's internal trace.
-    // Assume that frame is the invocation frame.
-    const declarationFrame = frames.slice(1).find((frame) => {
-      return !/(node_modules)?\/lib\/(umd|esm)\//.test(frame)
-    })
+  // Get the first frame that doesn't reference the library's internal trace.
+  // Assume that frame is the invocation frame.
+  const ignoreFrameRegExp = /(node_modules)?[\/\\]lib[\/\\](umd|esm)[\/\\]|^[^\/\\]*$/
```

Comment:
I might be wrong here, but shouldn't we also at the iife build here?
```suggestion
  const ignoreFrameRegExp = /(node_modules)?[\/\\]lib[\/\\](umd|esm|iife)[\/\\]|^[^\/\\]*$/
```

Reply:
@timdeschryver I think you're totally right. Ideally, we should include all the build targets (umd, esm, cjs, iife) so all source file references can be accounted for.

Reply:
- [ ] add `|iife|cjs` to regex